forge-gui/res/cardsfolder/upcoming/urabrask_the_great_work.txt

```diff
@@ -0,0 +1,36 @@
+Name:Urabrask
+ManaCost:2 R R
+Types:Legendary Creature Phyrexian Praetor
+PT:4/4
+K:First Strike
+T:Mode$ SpellCast | ValidCard$ Card.Instant,Card.Sorcery | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDealDamage | TriggerDescription$ Whenever you cast an instant or sorcery spell, CARDNAME deals 1 damage to target opponent.
+SVar:TrigDealDamage:DB$ DealDamage | ValidTgts$ Opponent | TgtPrompt$ Select target opponent | NumDmg$ 1 | SubAbility$ DBMana | SpellDescription$ Add {R}.
```

Comment:
Are you sure the full description gets built like that?

Reply:
Looks like the "Add {R}" was not displaying. Fixed it, thank you!